content/en/docs/api/allpackages-index.html

```diff
@@ -1,16 +1,15 @@
 <!DOCTYPE HTML>
 <html lang="en">
 <head>
-<!-- Generated by javadoc (17) on Thu Jan 23 16:25:28 GMT 2025 -->
-<title>All Packages (Narayana: narayana-full 7.2.0.Final API)</title>
+<!-- Generated by javadoc (21) on Fri Jan 31 10:43:11 CET 2025 -->
```

Comment:
I realise that a lot of the changes are dates and version numbers, I wonder if the different Javadoc tool version might account a bit for changes and maybe we should ask the release process users to use a specific JDK major version (I guess 21 is maybe good) for Javadoc generation?

Reply:
The date will change too, so we will still have the diff change. BTW in the wiki the release require a JDK >=17

Reply:
We could make the wiki process say to use JDK=21

Reply:
That is ok with me!